docs/changehistory/NextVersion.md

```diff
@@ -90,6 +91,16 @@ The new class [IndexedPolyfaceWalker]($core-geometry) has methods to complete th
 
 If a walker operation would advance outside the mesh (e.g., `edgeMate` of a boundary edge), it returns an invalid walker.
 
+### Text Block Margins
+
+A new enhancement has been made to the [TextBlock]($core-common) class to add support for margins. These margins are in meters, and are specified by using the [TextBlockMargin]($core-common) interface.
+
+The margin range is calculated in [TextAnnotationLayout.layoutTextBlock]($core-backend) and saved to the [TextBlockLayout.range]($core-backend) property. The range of just the text itself is stored in a property called [TextBlockLayout.textRange]($core-backend). In general, use of `range` is preferred over `textRange`.
```

Comment:
@ben-polinsky no docs build error despite no function called `TextAnnotationLayout.layoutTextBlock`. Is this some feature I'm not familiar with where you use the filename sans extension to disambiguate a free function's name?

Reply:
Resolving, but question for @ben-polinsky remains.